packages/model-viewer/src/features/scene-graph/image.ts

```diff
@@ -71,8 +73,13 @@ export class Image extends ThreeDOMElement implements ImageInterface {
     return this.uri != null ? 'external' : 'embedded';
   }
 
+  set name(name: string) {
+    (this[$sourceObject] as GLTFImage).name = "image_" + name;
```

Comment:
Do we want to prepend `image_` here? Seems odd to set one name and get something different back.

Reply:
Yep, this is a typo from development tests. Removed on next commit